app/interface/deps/client_deps.py

```
@@ -0,0 +1,31 @@
+from typing import Union
+from collections.abc import AsyncGenerator
+
+from fastapi import Body, HTTPException
+
+from app.interface.schemas.common.requests import LoginRequest
+from app.interface.clients import KutisClient, LmsClient, BarunClient
+from app.infrastructure.exceptions import ClientException
+
+
+def client_provider_factory(
```

Ruff (0.12.2)

Missing return type annotation for public function `client_provider_factory`

(ANN201)